browser/ui/views/commands/default_accelerators_mac.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_COMMANDS_DEFAULT_ACCELERATORS_MAC_H_
+#define BRAVE_BROWSER_UI_VIEWS_COMMANDS_DEFAULT_ACCELERATORS_MAC_H_
+
+#include <vector>
+
+#include "chrome/browser/ui/views/accelerator_table.h"
+
+namespace commands {
+
+// Returns shortcuts mapped to global menu, which can be changed dynamically
+// by the OS too. This is why we don't have a static table for these.
+// (System Preference > Keyboard > Keyboard Shortcuts > App shortcuts)
+std::vector<AcceleratorMapping> GetGlobalAccelerators();
```

Comment:
Out of curiosity (we don't need to handle it yet) can the shortcuts be changed at runtime?

Reply:
Yeah I think so. But I'm not sure if/how we can observe the change. 🤔